app/javascript/components/requests/RequestForm.jsx

```diff
@@ -82,10 +89,28 @@ class RequestForm extends React.Component {
     if (!this.props.buyer) {
       errors["login"] = "You must be logged in to request a work."
     }
-
+    if (this.state.exist) {
+      errors["exist"] = "You have already made a request under this request type."
+    }
     return errors;
   }
 
+  checkExist = (request_type) => {
+    let stringifiedSearchParams = [`buyer_id=${this.props.buyer.id}`, `artist_id=${this.props.artist_id}`,
```

Comment:
Might be a little cleaner if we destructure the props?
```suggestion
    const { buyer, artist_id, work_id } = this.props;
    let stringifiedSearchParams = [
      `buyer_id=${buyer.id}`,
      `artist_id=${artist_id}`,
      `work_id=${work_id}`,
      `types=${request_type}`
    ].join("&");
```